lib/gui/app.js

```diff
@@ -164,25 +187,38 @@ app.run(($window, WarningModalService, ErrorService, FlashStateModel, OSDialogSe
     // Don't open any more popups
     popupExists = true;
 
+    AnalyticsService.logEvent('Close attempt while flashing');
+
     return OSDialogService.showWarning({
       confirmationLabel: 'Yes, quit',
       rejectionLabel: 'Cancel',
       title: 'Are you sure you want to close Etcher?',
       description: messages.warning.exitWhileFlashing()
     }).then((confirmed) => {
       if (confirmed) {
+        AnalyticsService.logEvent('Close confirmed while flashing');
```

Comment:
Did you double check if this event is triggered correctly? We're killing the process right afterwards and it might be that this event has no change to be emitted.